enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/iverksett/IverksettDto.kt

```diff
@@ -102,6 +109,18 @@ data class VedtaksdetaljerBarnetilsynDto(
         val tilleggsstønad: List<PeriodeMedBeløpDto>
 ) : VedtaksdetaljerDto()
 
+data class VedtaksdetaljerSkolepengerDto(
+        override val resultat: Vedtaksresultat,
+        override val vedtakstidspunkt: LocalDateTime,
+        override val opphørÅrsak: OpphørÅrsak?,
```

Comment:
Er opphør relevant for skolepenger?

Reply:
Jeg og Eirik ble enige om at vi beholder denne tils videre

Reply:
Diskuterte denne og ble enige om at vi beholder den.